src/main/java/com/github/fabienrenaud/jjb/stream/UsersStreamSerializer.java

```diff
@@ -1149,4 +1156,90 @@ public String underscore_java(final Users obj) throws IOException {
         }
         return jso;
     }
+
+    @Override
+    public Value purejson(Users obj) throws IOException {
+        final Map<Parser.Value, Parser.Value> map = new LinkedHashMap<>();
+        final List<Parser.Value> arr = new ArrayList<>();
+        if (obj.users != null) {
+            for (User u : obj.users) {
+                arr.add(purejson(u));
+            }
+            map.put(Value.string("users"), Value.arr(arr));
+        }
+        return Value.json(map);
+    }
+    
+    private Parser.Value purejson(final User u) throws IOException {
+        final Map<Parser.Value, Parser.Value> map = new LinkedHashMap<>();
+        
+        if (u._id != null) {
+            map.put(Value.string("_id"), Value.string(u._id));
+        }
+        map.put(Value.string("index"), Value.integer(u.index));
```

Comment:
note: having to manually box all the values is boiler-plate and not developer-friendly. I suggest you overload your API methods to support std java types...